homeassistant/components/deconz/alarm_control_panel.py

```diff
@@ -55,7 +55,7 @@
 
 def get_alarm_system_for_unique_id(gateway, unique_id: str):
     """Retrieve alarm system unique ID is registered to."""
-    for alarm_system in gateway.api.alarm_systems.values():
+    for alarm_system in gateway.api.alarmsystems.values():
```

Comment:
This one seemed a bit inconsistent, but I verified the change matches the upstream one.

Reply:
Yeah thats the only place that went the other way :), thorough readthrough Nick!